doc/oper.xml

```diff
@@ -1352,8 +1352,8 @@ false
 
 <#GAPDoc Label="VerticesReachableFrom">
 <ManSection>
-  <Oper Name="VerticesReachableFrom" Arg="digraph, root"/>
-  <Oper Name="VerticesReachableFrom" Arg="digraph, list"/>
+  <Oper Name="VerticesReachableFrom" Arg="digraph, root" Label="for a vertex"/>
+  <Oper Name="VerticesReachableFrom" Arg="digraph, list" Label="for a list of vertices"/>
```

Comment:
Should the labels include "a digraph and"?

Reply:
Good catch, updating this now.